stubs/seaborn/seaborn/regression.pyi

```diff
@@ -1,19 +1,22 @@
 from _typeshed import Incomplete
-from collections.abc import Iterable
-from typing import Any
-from typing_extensions import Literal
+from collections.abc import Callable, Iterable
+from typing import Any, overload
+from typing_extensions import Literal, TypeAlias
 
 import pandas as pd
 from matplotlib.axes import Axes
 from matplotlib.typing import ColorType
+from numpy.typing import NDArray
 
 from .axisgrid import FacetGrid
 from .utils import _Palette, _Seed
 
 __all__ = ["lmplot", "regplot", "residplot"]
 
+_Vector: TypeAlias = list[Incomplete] | pd.Series[Incomplete] | pd.Index[Incomplete] | NDArray[Incomplete]
```

Comment:
`_Vector` is defined according to [this function](https://github.com/mwaskom/seaborn/blob/d4b8de81f74a153bb3c15af760d956d32aad980b/seaborn/regression.py#L32-L54). All parameters annotated with `_Vector` are handled by this function